TrafficCapture/trafficCaptureProxyServer/src/test/java/org/opensearch/migrations/trafficcapture/proxyserver/netty/NettyScanningHttpProxyTest.java

```diff
@@ -187,12 +187,15 @@ private static String makeTestRequestViaClient(SimpleHttpClientForTesting client
             nshp.set(new NettyScanningHttpProxy(port.intValue()));
             try {
                 URI testServerUri = new URI("http", null, SimpleHttpServer.LOCALHOST, underlyingPort,
-                    null, null, null);
+                        null, null, null);
```

Comment:
nit: replace tab

Reply:
This is spaces - the current style that this project is using favors a massive amount of indentation (too much for my liking - but something for another day).